docs/tutorials/integrations/nginx/tls/tls.conf

```diff
@@ -0,0 +1,39 @@
+[req]
+default_bits = 2048
+prompt = no
+default_md = sha256
+req_extensions = req_ext
+distinguished_name = dn
+x509_extensions = v3_ca # The extentions to add to the self signed cert
+req_extensions  = v3_req
+x509_extensions = usr_cert
+
+[ dn ]
+C=US
+ST=Wisconsin
+L=Madison
+O=Onyx
+OU=CyberArk
+CN=proxy
+
+[ usr_cert ]
+basicConstraints=CA:FALSE
+nsCertType                      = client, server, email
+keyUsage = nonRepudiation, digitalSignature, keyEncipherment
+extendedKeyUsage = serverAuth, clientAuth, codeSigning, emailProtection
```

Comment:
codeSigning is definitely not needed here; cf. above for client and email.

Reply:
I gave some effort to this and other recommendations for the TLS conf, but was only able to make the DN changes & keep the cert working. Let's take a second pass at this later and optimize/simplify the tls.conf.

Reply:
Ok, can you create an issue?